packages/playground/src/components/select_farm.vue

```diff
@@ -169,20 +167,11 @@ async function loadFarms() {
   }
   farms.value = farms.value.concat(_farms);
 
-  if (oldFarm) {
-    farm.value = undefined;
-    await nextTick();
-    farm.value = farms.value.find(f => f.name === oldFarm.name);
+  farm.value = farms.value[0];
+  farmInput.value.setStatus(initialized ? ValidatorStatus.Invalid : ValidatorStatus.Init);
```

Comment:
It looks like this caused an error.
network: main 
![image](https://github.com/threefoldtech/tfgrid-sdk-ts/assets/62248851/4462f44a-c47c-4de4-b250-bdab331acf79)